src/settings/AudioSettings.tsx

```diff
@@ -125,20 +134,41 @@ export default function GeneralSettings(props: ISettingsPanelProps) {
       color: '#bb4220',
     },
   };
+  const switchStyle = {
```

Comment:
I'd love to pick your brain on how you came up with this sometime. This still looks like a foreign language to me. I've copied loads of similar things of SO and not understood any of it.

Reply:
Sure! I was referencing some MUI docs to come up with it but as I understand it it's CSS passed as an object. There should be an easier way to do this through theming - I'll message you about that